contrib/cni/README.md

```diff
@@ -5,11 +5,31 @@ directory just contains some example configurations that can be used as the
 basis for your own configurations (distributions should package these files in
 example directories).
 
-To use these configurations, place them in `/etc/cni/net.d` (or the directory
-specified by `crio.network.network_dir` in your overwrite of `/etc/crio/crio.conf.d`).
+### Configuration Directory
 
-CRI-O will only choose the network that has been specified via
-`crio.network.cni_default_network`.
+By default, your CNI configurations are read from `/etc/cni/net.d`.
+This can be overwritten by specifying `crio.network.network_dir` in your overwrite of `/etc/crio/crio.conf.d`
+
+CRI-O chooses a CNI configuration from this directory with lexicographic precidence (10-config will be chosen over 99-config).
+However, CRI-O will only choose a network whose name matches the value of `crio.networkcni_default_network` (default value is `crio`).
+
+Unless you have a specific networking configuration you'd like to use, we recommend installing either [10-crio-bridge.conf][dual-stack], or [11-crio-ipv4-bridge.conf][ipv4-only]
+
+By default, we install the dual stack version: [10-crio-bridge.conf][dual-stack]
+
+However, if you are installing on a node with ipv6 disabled (`sysctl net.ipv6.conf.default.disable_ipv6` and `sysctl net.ipv6.conf.all.disable_ipv6` == 0)
+then we recommend you install the ipv4 only version: [11-crio-ipv4-bridge.conf][ipv4-only]
+Otherwise, you'll run into an error similar to:
+```
+Interface vetha38a080a Mac doesn't match: ee:7b:4d:57:3a:d9 not found
```

Comment:
I would say that this is a CNI plugin issue.

Reply:
I would agree, and there are other cases that cause it too, but I think the workaround is worth documenting

Reply:
Yeah, I am fine with that, but let's open an issue with ocicni/cni to track.